### PR TITLE
Update allowanon.md

### PR DESCRIPTION
**Pull request by AndriiMaliuta**
Kotlin language example added

## Description:
Kotlin language example added to allowanon.md

### CODE REVIEW

**Comment by AndriiMaliuta** [comment]
> @AndriiMaliuta To make this information appear, you need to add a stackselector in the `index.md` file that holds the common content, similar to:
> 
> `<StackSelector snippet="allowanon.md"`
> 
> Alternatively, you may need to add this content to an existing snippet file (i.e., either reqautheverthhing.md or reqauthspecfic.md). If you need assistance, let me know. @andri

@brianduffield-okta , I added it to the **"reqautheverthhing.md"** file

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md [hunk 1]
```diff
@@ -19,3 +20,31 @@ class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapte
     }
 }
 ```
+
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md:23]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md [hunk 2]
```diff
@@ -19,3 +20,31 @@ class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapte
...
+
+Kotlin:
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md:25]:
```suggestion
Kotlin:

```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md [hunk 3]
```diff
@@ -19,3 +20,31 @@ class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapte
...
+                authorize("/my-anon-page", permitAll)
+                authorize( anyRequest, authenticated)
+            }
+            oauth2Login {  }
+        }
+
+    }
+
+}
+
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md:46]: You need to add the close code-block (```) on line 46.

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md [hunk 4]
```diff
@@ -19,3 +20,31 @@ class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapte
...
+
+
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md:48]:
```suggestion
```

**Review by brianduffield-okta — COMMENTED** [review]
Just some minor formatting and line spacing plus the addition of (```) to close the code block.

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md (hunk 3 above)
Comment (AndriiMaliuta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md]: Hello @brianduffield-okta ,
Added the required changes.
Thanks!

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md]: @AndriiMaliuta Thanks for your efforts on this update, Andrii. I currently have a few questions out to our Developer teams regarding this addition and hope to hear back shortly.

**Comment by brianduffield-okta** [comment]
Closing this PR. These additions were merged with #2818 Thank you, @AndriiMaliuta

**Pull request closed without merging**